ApplicationLibCode/ProjectDataModel/Flow/RimWellAllocationOverTimeCollection.cpp

```diff
@@ -81,6 +81,22 @@ RimWellAllocationOverTimeCollection::RimWellAllocationOverTimeCollection( const
     m_wellValuesMap = m_defaultWellValuesMap;
 }
 
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+void RimWellAllocationOverTimeCollection::fillWithFlowRateFractionValues()
```

Comment:
Rename `RimWellAllocationOverTimeCollection` to `RigWellAllocationOverTime`, and consider moving to Rig-location. No Rim-classes are used here, and the intention is to use Rig-prefix for data classes.
This was not detected in previous PR.